migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc

```diff
@@ -32,4 +32,4 @@ xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#advanced-m
 * Enabling automatic PV resizing for direct volume migration
 * Enabling cached Kubernetes clients for improved performance
 
-For new features and enhancements, technical changes, and known issues, see the xref:../migration-toolkit-for-containers/mtc-release-notes.adoc#mtc-release-notes[{mtc-short} release notes].
+For new features and enhancements, technical changes, and known issues, see the xref:../migration_toolkit_for_containers/mtc-release-notes.adoc#mtc-release-notes[{mtc-short} release notes].
```

Comment:
Why did this need to be fixed? The xref is correct in 'main'.

Reply:
This xref didn't work in the build, and the PR didn't pass the checks because of that.

Reply:
Of course it would not work in the build. But if you rebased your branch on 'main', the xref would be the correct one.

Reply:
But I did rebase it.

Reply:
Then you got a merge conflict and did not resolve it correctly. Look at 'main' and then look at your branch. You will see that 'main' has the correct xref, so it must be the same in your branch if it is correctly rebased. There is no alternative explanation.

Reply:
I rebased my branch before I made my changes. Then I made the changes and committed them. Then I saw that the checks failed because of that xref. And indeed, the xref didn't work in the build (although it did work in my local asciibinder build). Then I changed the xref, and it worked.